subprojects/core/src/test/groovy/org/gradle/execution/plan/ExecutionNodeAccessHierarchyTest.groovy

```diff
@@ -172,6 +173,31 @@ class ExecutionNodeAccessHierarchyTest extends Specification {
         assertNodesAccessing("/other", root)
     }
 
+    def "can query for nodes accessing using excludes for parts of paths"() {
```

Comment:
the test name reads strange.
Suggestion: "can return nodes accessing some path taking includes and excludes into consideration"

Reply:
Changed.